src/parser/jobs/mnk/modules/Fists.tsx

```diff
@@ -159,7 +196,17 @@ export default class Fists extends Module {
 			value: this.getFistGCDCount(STATUSES.FISTS_OF_EARTH.id),
 		}))
 
-		// TODO: Add check for FoF GCD counts, per window, being larger than expected
+		this.suggestions.add(new TieredSuggestion({
+			icon: ACTIONS.FISTS_OF_WIND.icon,
+			content: <Trans id="mnk.fists.suggestions.fow.content">
+				Avoid swapping to <StatusLink {...STATUSES.FISTS_OF_WIND}/> while below {MAX_STACKS} stacks until you're about to execute a <StatusLink {...STATUSES.COEURL_FORM} /> skill. <StatusLink {...STATUSES.FISTS_OF_FIRE} /> offers more damage until you can get to GL{MAX_FASTER}.
```

Comment:
Should this read "...while below X stacks **unless** you're about to..."? The "until" reads weirdly in my head.